scripts/bump-version.js

```diff
@@ -1,3 +1,11 @@
+#!/usr/bin/env node
+/**
+ * @fileoverview
+ * Update the simple-icons dependency to the latest version and update the
```

Comment:
We can avoid the repeat?

```suggestion
 * Update the simple-icons dependency to the latest version and the
```